plugins/chat/app/services/chat/auto_join_channels.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+module Chat
+  class AutoJoinChannels
```

Comment:
It doesn't seem like we have a test for this service class. Is there a reason we opted not to test it?